modules/test/playwright/tests/account-admin-web/accountGroups.spec.ts

```diff
@@ -66,3 +66,65 @@ test('LPD-25948 Can search for account group', async ({
 		0
 	);
 });
+
+test('LPD-28159 Account groups are displayed in account details page', async ({
+	accountsPage,
+	apiHelpers,
+	page,
+}) => {
+	const account = await apiHelpers.headlessAdminUser.postAccount({
+		name: getRandomString(),
+		type: 'business',
+	});
+
+	apiHelpers.data.push({id: account.id, type: 'account'});
+
+	const accountGroup1 = await apiHelpers.headlessAdminUser.postAccountGroup({
+		name: getRandomString(),
+	});
+
+	apiHelpers.data.push({id: accountGroup1.id, type: 'accountGroup'});
+
+	const accountGroup2 = await apiHelpers.headlessAdminUser.postAccountGroup({
+		name: getRandomString(),
+	});
+
+	apiHelpers.data.push({id: accountGroup2.id, type: 'accountGroup'});
+
+	const accountGroup3 = await apiHelpers.headlessAdminUser.postAccountGroup({
+		name: getRandomString(),
+	});
+
+	apiHelpers.data.push({id: accountGroup3.id, type: 'accountGroup'});
+
+	await apiHelpers.headlessAdminUser.assignAccountToAccountGroup(
```

Comment:
because you have to touch the test can you move this after the creation? 
so create1 + data push1 + assign1 / create2 + data push2 + assign2 ...

Reply:
@brbalazs is fine, just very little things